utils/dashboard.js

```diff
@@ -28,77 +28,71 @@ const dashboard = async (req, res) => {
     let userEmail = SSOObject.email;
     let siteDetails = SSOObject.siteDetails;
 
-    // Auth success, and proceed below steps:
     const { isParentEntity } = require('./shared');
     const authObj = await isParentEntity(siteDetails);
-    if(userEmail) authObj['userEmail'] = userEmail;
+    if (userEmail) authObj['userEmail'] = userEmail;
     const isParent = authObj.isParent;
     const siteCodes = authObj.siteCodes;
     const isCoordinatingCenter = authObj.coordinatingCenter;
     const isHelpDesk = authObj.helpDesk;
     const api = req.query.api;
     console.log(`SMDB API: ${api}, accessed by: ${userEmail}`);
 
-    if(api === 'validateSiteUsers') {
-        const { validateSiteUsers } = require('./validation');
-        return await validateSiteUsers(req, res, authObj);
```

Comment:
if we truly don't need call to `validateSiteUsers()` anymore, function should be removed as it doesn't appear to be called anywhere else